crates/turborepo-ci/src/lib.rs

```diff
@@ -0,0 +1,196 @@
+#![deny(clippy::all)]
+
+mod vendors;
+
+use std::env;
+
+use lazy_static::lazy_static;
+
+pub use crate::vendors::Vendor;
+use crate::vendors::VENDORS;
+
+lazy_static! {
```

Comment:
Can we use once_cell here instead? Saves a dep. Could use a function to get or initialize the value. In the long term we could use lazy_cell but we can wait until stabilized.